src/Microsoft.Data.SqlClient/netcore/src/Common/src/Microsoft/Data/ProviderBase/DbConnectionFactory.cs

```diff
@@ -45,38 +50,60 @@ protected DbConnectionFactory()
 
         public void ClearAllPools()
         {
-            Dictionary<DbConnectionPoolKey, DbConnectionPoolGroup> connectionPoolGroups = _connectionPoolGroups;
-            foreach (KeyValuePair<DbConnectionPoolKey, DbConnectionPoolGroup> entry in connectionPoolGroups)
+            long scopeID = SqlClientEventSource.Log.ScopeEnterEvent("<prov.DbConnectionFactory.ClearAllPools|{0}>", "API");
```

Comment:
I don't think you intended to do these changes in this PR, please apply your commit on "master" branch than Eventsource branch for Netcore changes.